gdml-generator/gdml-gen.cc

```diff
@@ -296,6 +303,35 @@ int main(int argc, char* argv[])
             gdml_filename = "thin-slab.gdml";
             break;
 
+        case GeometryID::simple_lz:
+            if (argc == 2)
+            {
+                run_manager->SetUserInitialization(new SimpleLZ());
+                gdml_filename = "simple_lz.gdml";
+                break;
+            }
+            else if (argc == 3)
+            {
+                int sqrt_num_pmts = std::atoi(argv[2]);
+                if (sqrt_num_pmts < 1)
+                {
+                    std::cout
+                        << "The sqrt_num_pmts parameter must be positive "
+                        << std::endl;
+                    return EXIT_FAILURE;
+                }
+                run_manager->SetUserInitialization(new SimpleLZ(sqrt_num_pmts));
+                gdml_filename = "simple_lz.gdml";
```

Comment:
Guess you could move the filename definition one scope up, just after `case GeometryID::simple_lz`, so it is called once and easier to rename.